provider/pact-jvm-provider/src/main/groovy/au/com/dius/pact/provider/ProviderVerifier.groovy

```diff
@@ -1,13 +1,16 @@
 package au.com.dius.pact.provider
 
+import au.com.dius.pact.core.model.BrokerUrlSource
 import au.com.dius.pact.core.model.FilteredPact
 import au.com.dius.pact.core.model.Interaction
 import au.com.dius.pact.core.model.Pact
 import au.com.dius.pact.core.model.PactReader
+import au.com.dius.pact.core.model.PactSource
 import au.com.dius.pact.core.model.UrlPactSource
 import au.com.dius.pact.core.pactbroker.PactBrokerClient
 import au.com.dius.pact.core.pactbroker.TestResult
 import groovy.util.logging.Slf4j
+import org.jetbrains.annotations.Nullable
```

Comment:
This is a Jetbrains annotation, but I don't think the library is included anywhere. Unless it comes along with Kotlin?

Reply:
I'm not sure myself. I copied it from https://github.com/DiUS/pact-jvm/blob/master/provider/pact-jvm-provider/src/main/groovy/au/com/dius/pact/provider/reporters/JsonReporter.groovy#L16

Reply:
Hmm, I think IntelliJ must have slipped that one in. But it works ok, so all good.